runner/providers/v0.1.1/util.go

```diff
@@ -0,0 +1,21 @@
+package v011
+
+import (
+	commonParams "github.com/cloudbase/garm-provider-common/params"
+)
+
+// IsProviderValidStatus checks if the given status is valid for the provider.
+// A provider should only return a status indicating that the instance is in a
+// lifecycle state that it can influence. The sole purpose of a provider is to
+// manage the lifecycle of an instance. Statuses that indicate an instance should
+// be created or removed, will be set by the controller.
+func IsValidProviderStatus(status commonParams.InstanceStatus) bool {
```

Comment:
This can probably live in a common package considering this validates what GARM expects from any provider (external or otherwise)

Reply:
Created a new `util` package in `runner/providers/util` for this common part